cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -22,18 +22,13 @@ import (
 	"github.com/adevinta/vulcan-check-sdk/helpers"
 	checkstate "github.com/adevinta/vulcan-check-sdk/state"
 	"github.com/avast/retry-go"
-	"github.com/mcuadros/go-version"
 )
 
-const (
-	vulnTruncateLimit   = 30
-	vulnCVETrucateLimit = 10
-)
+const vulnCVETrucateLimit = 10
```

Comment:
With the expansion of vulnerabilities now is true that we don't need to limit the number of vulnerabilities to 30. Because of this I would suggest to increase the number of `vulnCVETrucateLimit` to 30 instead.
As extra context, we would take the list of CVEs in order to compile the fingerprint of the vulnerability and as less information we loose the better.

Reply:
Ok, but we could keep the list in the table to 10 and use the full list of CVEs for the finterprint.

Reply:
Kept the 10 limit and using the whole list for the fingerprint.